### PR TITLE
Troubleshooting guides for disabled allocations

### PR DESCRIPTION
**Pull request by andreidan**
This adds the troubleshooting guides when index and cluster allocations are
disabled.

### CODE REVIEW

File: docs/reference/tab-widgets/troubleshooting/data/enable-index-allocation.asciidoc [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+      "index.routing.allocation.enable": "none" <1>
+    }
+  }
+}
+----
+// TESTRESPONSE[skip:the result is for illustrating purposes only]
+
++
+<1> Represents the current configured value that controls if the index is allowed 
+to partially or totally allocate.
```
Comment (dakrone) [inline docs/reference/tab-widgets/troubleshooting/data/enable-index-allocation.asciidoc:73]:
```suggestion
to be partially or totally allocated.
```

File: docs/reference/tab-widgets/troubleshooting/data/enable-index-allocation.asciidoc [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
```
Comment (dakrone) [inline docs/reference/tab-widgets/troubleshooting/data/enable-index-allocation.asciidoc:125]:
```suggestion
to be partially or totally allocated.
```

File: docs/reference/troubleshooting/data/enable-cluster-allocation.asciidoc [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+== Allow Elasticsearch to allocate the data in the system
+
+The allocation of data in an {es} deployment can be controlled using the <<cluster-routing-allocation-enable, enable 
+cluster allocation configuration>>.
+In certain circumstances users might want to temporarily disable or restrict the
+allocation of data in the system. 
+
+Forgetting to re-allow all data allocations can lead to unassigned shards.
+
+In order to (re)allow all data to be allocated follow the next steps:
```
Comment (dakrone) [inline docs/reference/troubleshooting/data/enable-cluster-allocation.asciidoc:11]:
```suggestion
In order to (re)allow all data to be allocated follow these steps:
```

File: docs/reference/troubleshooting/data/enable-index-allocation.asciidoc [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+== Allow Elasticsearch to allocate the index
+
+The allocation of data can be controlled using the <<index-routing-allocation-enable-setting, enable 
+allocation configuration>>.
+In certain circumstances users might want to temporarily disable or restrict the
+allocation of data. 
+
+Forgetting to re-allow all data allocation can lead to unassigned shards.
+
+In order to (re)allow all data to be allocated follow the next steps:
```
Comment (dakrone) [inline docs/reference/troubleshooting/data/enable-index-allocation.asciidoc:11]:
```suggestion
In order to (re)allow all data to be allocated follow these steps:
```

**Review by dakrone — APPROVED** [review]
I left some really minor comments, but LGTM otherwise!

File: docs/reference/tab-widgets/troubleshooting/data/enable-cluster-allocation.asciidoc [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+. Log in to the {ess-console}[{ecloud} console].
++
+
+. On the **Elasticsearch Service** panel, click the name of your deployment. 
++
+
+NOTE:
+If the name of your deployment is disabled your {kib} instances might be
+unhealthy, in which case please contact https://support.elastic.co[Elastic Support],
+or your deployment doesn't include {kib}, in which case all you need to do is
```
Comment (jbaiera) [inline docs/reference/tab-widgets/troubleshooting/data/enable-cluster-allocation.asciidoc:22]: Same as other PR about run on sentence.

File: docs/reference/tab-widgets/troubleshooting/data/enable-index-allocation.asciidoc [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+. Log in to the {ess-console}[{ecloud} console].
++
+
+. On the **Elasticsearch Service** panel, click the name of your deployment. 
++
+
+NOTE:
+If the name of your deployment is disabled your {kib} instances might be
+unhealthy, in which case please contact https://support.elastic.co[Elastic Support],
+or your deployment doesn't include {kib}, in which case all you need to do is
```
Comment (jbaiera) [inline docs/reference/tab-widgets/troubleshooting/data/enable-index-allocation.asciidoc:40]: "run on sentence" advice part II: Split sentence at "or", start next sentence with "if", etc...

**Review by jbaiera — APPROVED** [review]
LGTM pending small edits

**Pull request merged**